site/docs/components/toggle-button/examples.mdx

```diff
@@ -0,0 +1,80 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="toggle-button" exampleName="ToggleButtonDefault" displayName="Toggle Button">
```

Comment:
```suggestion
<LivePreview componentName="toggle-button" exampleName="ToggleButtonDefault">
```

Reply:
`displayName` is needed as we would like the 'select an example' left side panel to read "Toggle Button" instead of "Toggle Button Default"